Utilities/bootstrap

```diff
@@ -309,8 +322,11 @@ def build(args):
         build_swift_argument_parser(args)
         build_swift_driver(args)
         build_swift_crypto(args)
+        build_swiftpm_with_cmake(args)
+
+    if args.cross_compile_flags:
+        build_packagedescription_libs_with_cmake(args)
 
-    build_swiftpm_with_cmake(args)
```

Comment:
Rather than reuse this function to cross-compile the PackageDescription libraries, I extracted that portion into `build_packagedescription_libs_with_cmake()` above and this function is only used to natively compile SPM with CMake now.